Gems/AudioSystem/Code/Include/Engine/AudioLogger.h

```diff
@@ -15,13 +15,13 @@
 namespace Audio
 {
     ///////////////////////////////////////////////////////////////////////////////////////////////////
-    enum EAudioLogType
+    enum class LogType
```

Comment:
I'm curious why you make a special logger for audio that redirects to the AZ_Trace functions. Why not just use the ILogger interface from AzCore? It already has wrappers for AZLOG_INFO, AZLOG_NOTICE, AZLOG_WARN, AZLOG_ERROR, etc.. using those macros from AzCore means it will also pipe to the imgui console log.

Reply:
Frankly, I'd like to get rid of this and use a general log system.  I think originally a special wrapper around audio logs was created so it could have its own verbosity setting.  Sometimes audio logs can get quite spammy at higher verbosities, or when you're trying to tune/debug/tweak some functionality.  If log groups/tags are supported in `ILogger` (and it looks like it might be) then I can just change everything to use `AZLOG`.

Reply:
Went ahead and removed AudioLogger and replaced log lines with `AZLOG*`